resources/lang/lt/errors.php

```diff
@@ -0,0 +1,4 @@
+<?php
+return [
```

Comment:
Please add breakline before return.

```suggestion

return [
```